iOS/DuckDuckGo/HomeMessageViewModel.swift

```diff
@@ -141,7 +141,8 @@ struct HomeMessageViewModel {
             }
         case .survey(let value):
             return { @MainActor in
-                LaunchTabNotification.postLaunchTabNotification(urlString: value)
+                let refreshedUrl = refreshLastSearchState(in: value)
```

Comment:
Force a refresh of last_search_state when starting the survey in case user has performed a search since the RMF message was presented